src/test_utils/mod.rs

```diff
@@ -257,6 +273,18 @@ pub(crate) fn deploy_evm() -> AuroraRunner {
 
     assert!(maybe_error.is_none());
 
+    let args = InitCallArgs {
+        prover_account: "prover.near".to_string(),
+        eth_custodian_address: "d045f7e19B2488924B97F9c145b5E51D0D895A65".to_string(),
+    };
+    let (_, maybe_error) = runner.call(
+        "new_eth_connector",
```

Comment:
This is the new call I mentioned in the PR description is needed to make the Engine usable. It makes me wonder if we should roll it in with `new` (why have two calls if they are both required?). cc @mrLSD @artob